mysql-test/mroonga/storage/create/table/index/parser/t/comment.test

```diff
@@ -16,6 +16,10 @@
 
 --source ../../../../../../include/mroonga/not_embedded.inc
 --source ../../../../../../include/mroonga/have_mroonga.inc
+# This test is MySQL-only because MariaDB's handling of the deprecated 'parser'
+# syntax has changed over time. Different MariaDB versions show different
+# warning messages or automatically convert 'parser' to 'tokenizer'.
+--source ../../../../../../include/mroonga/have_mysql.inc
```

Comment:
We should use `have_mysql.inc` before `have_mroonga.inc` to avoid unbalanced `have_mroonga.inc` and `have_mroonga_deinit.inc`.

Reply:
fix: [942ec44](https://github.com/mroonga/mroonga/pull/941/commits/942ec444545a58d9d0519a8863faeed5f6e5e7b5) Thanks!